main/webapp/modules/core/scripts/dialogs/clustering-dialog.js

```diff
@@ -387,7 +415,8 @@ ClusteringDialog.prototype._updateData = function(data) {
             edit: false,
             choices: this,
             value: this[0].v,
-            size: this.length
+            size: this.length,
+            checkBoxs : []
```

Comment:
```suggestion
            checkBoxes : []
```
(which also would need updating where this attribute is referenced)